src/connector/connector_utils.go

```diff
@@ -948,3 +948,26 @@ func RemoveRoCEDevice(device string) ([]string, string, error) {
 
 	return devices, multiPathName, nil
 }
+
+// RemoveNvmeFcDevice remove dm device if present
+func RemoveNvmeFcDevice(device string) (string, error) {
+	var multiPathName string
+	var err error
+	if strings.HasPrefix(device, "dm") {
```

Comment:
This is valid only for DM multipath. If other multipath like Ultrapath or EMCPP comes then we will have to put other checks